ytmusicapi/mixins/_protocol.py

```diff
@@ -17,15 +18,21 @@ class MixinProtocol(Protocol):
 
     proxies: Optional[dict[str, str]]
 
+    context: dict
+
     def _check_auth(self) -> None:
         """checks if self has authentication"""
+        ...
 
     def _send_request(self, endpoint: str, body: dict, additionalParams: str = "") -> dict:
         """for sending post requests to YouTube Music"""
+        ...
```

Comment:
The dots are not needed, please remove?

Reply:
Pylance will report an error if not. 
(if you would remove the docstring, it would throw an error anyway, as the function is technically empty. See: https://docs.python.org/3/library/typing.html#typing.overload)

Reply:
This project uses mypy, so please don't worry about pylance findings. For mypy it's ok to have a docstring in place of `...`

It's a matter of personal preference - for me the docstring has a purpose and fulfills the requirement of no empty function body, so I like it better than the nondescript `...`

Reply:
Side note: I have triggered the lint jobs so you can see the mypy and ruff findings remaining. 

I recommend taking a look at https://github.com/sigma67/ytmusicapi/blob/main/CONTRIBUTING.rst

where pre-commit is described - if that passes, the lint jobs should pass as well.